pkgs/by-name/we/webrtc-audio-processing_2/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  fetchurl,
+  darwin,
+  abseil-cpp,
+  meson,
+  ninja,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

`rec` with a large range was considered as an anti-pattern